googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/resources/artifacts/ESB/config/proxies/googleprediction/googleprediction.xml

```diff
@@ -1,84 +1,113 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fixed in https://github.com/wso2-extensions/esb-connector-googleprediction/pull/2